core/objects.go

```diff
@@ -144,6 +144,10 @@ type ValidationRecord struct {
 	//   ...
 	// }
 	AddressesTried []net.IP `json:"addressesTried,omitempty"`
+	// ResolverAddresses is typically the host:port of the DNS resolver(s) that
+	// fulfilled the lookup for AddressUsed. During recursive A and AAAA
+	// lookups, a record may look like A:host:port or AAAA:host:port
+	ResolverAddresses []string `json:"resolverAddresses,omitempty"`
```

Comment:
nit: We're already using the name "ResolverAddrs" elsewhere in this PR, and this is the place where we actually determine the number of bytes that get chucked into the database every time we store a validation record as json. So, tiny optimization:
```suggestion
	ResolverAddrs []string `json:"resolverAddrs,omitempty"`
```